packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +155,8 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    ++this.readStackEventsCalls;
```

Comment:
I think its better to use exponential backoff to create min api requests. 

https://docs.aws.amazon.com/general/latest/gr/api-retries.html